.pre-commit-config.yaml

```diff
@@ -1,14 +1,14 @@
 repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.0.1
+  rev: v4.1.0
   hooks:
     - id: trailing-whitespace
     - id: end-of-file-fixer
     - id: check-yaml
     - id: check-added-large-files
     - id: mixed-line-ending
 - repo: https://github.com/gruntwork-io/pre-commit
-  rev: v0.1.16
+  rev: v0.1.17
   hooks:
     - id: shellcheck
     - id: gofmt
```

Comment:
Please note that there is a hadolint hook also for pre-commit. Should we delete the one in Github actions and add it there?

Reply:
hadolint is not present in actions. it has only dot-file. every linter with hadolint support will apply only the suggested flags